openhexa.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=%i service
+PartOf=docker.service network-online.target
+After=docker.service network-online.target
+AssertPathExists=/usr/local/openhexa/compose.yml
+
+[Service]
+Type=oneshot
+RemainAfterExit=true
+WorkingDirectory=/etc/openhexa
+ExecStart=/usr/bin/docker compose --file /usr/local/openhexa/compose.yml --project-name openhexa --profile frontend --profile minio --profile pipelines up -d --remove-orphans
```

Comment:
Shouldn't we use `start` instead of `up` to avoid to use newer images by inadvertance?